doc/specs/stdlib_hashmaps.md

```diff
@@ -1148,6 +1154,9 @@ Procedures to modify the content of a map:
 
 Procedures to report the content of a map:
 
+* `map % get_all_keys( all_keys )` - Returns all the keys
+  presented in the map;
```

Comment:
Should it be "present" or "presented"?

Reply:
I use "presented in" to mean "located in," "existing in," or "contained in" here.
I cannot answer whether "present" or "presented" is correct due to my poor English skills, so I need help to create a natural expression, especially a phrase referring to "keys in a hash map."

Reply:
@milancurcic Any idea how it should be written?

Reply:
@jvdp1 
The following expression is used in the [documentation](https://docs.oracle.com/javase/8/docs/api/java/util/Map.html) of Map in Java:
> keySet()
>Returns a Set view of the keys contained in this map.

So, I would like to change "presented" to "contained" to follow this expression. I would appreciate your comments.
Thank you.

Reply:
@degawa "contained" sounds good to me. Please, go ahead!

Reply:
Thank you, @jvdp1, for giving me a supportive push forward.
I fixed the documentation for the `get_all_keys` and pushed it.
I would appreciate a review.